CPP Chat/ServerClass.cpp

```diff
@@ -40,28 +39,50 @@ SOCKET ServerClass::InitializeSocket()
         std::cout << "Can't create a socket! Quitting\n";
         return INVALID_SOCKET;
     }
+    std::cout << "Socket Initialized!\n";
     return listeningSocket;
 }
 
 fd_set ServerClass::CreateSocketSet()
```

Comment:
i don't think you use this function.
even so, you can make it initialize your socket as a class member